### PR TITLE
ZEN-32461: Add env variables to mariadb services

### PR DESCRIPTION
**Pull request by dlysak**

### CODE REVIEW

File: services/Zenoss.cse/Infrastructure/mariadb-events/service.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
     "Context": {
         "max_connections": "1000"
     },
+    "Environment": [
+        "MARIADB_DB=zep"
```
Comment (jpeacock-zenoss) [inline services/Zenoss.cse/Infrastructure/mariadb-events/service.json]: Is "zep" the correct name?  I think the name of zeneventserver's database is "zenoss_zep".  At least that's what I see in the database creation script in product-assembly.

Comment (dlysak) [reply services/Zenoss.cse/Infrastructure/mariadb-events/service.json]: this variable is used for this migration script which was checked and merged before in the context of the ticket https://github.com/zenoss/zenoss-prodbin/pull/3827/files#diff-55b141d4a13e67d7cdad4691d9c3a299R29-R30

Comment (dlysak) [reply services/Zenoss.cse/Infrastructure/mariadb-events/service.json]: we needed to get the migration result when creating a new instance

Comment (dlysak) [reply services/Zenoss.cse/Infrastructure/mariadb-events/service.json]: also, we have updated quiesce-mariadb.sh script with these names here: https://github.com/zenoss/zenoss-prodbin/pull/3766/files

Comment (jpeacock-zenoss) [reply services/Zenoss.cse/Infrastructure/mariadb-events/service.json]: All the migrate script does is add the variable to the service definition if it's not already there.

I found it.  It's used by the quiesce-mariadb.sh script.  The MARIADB_DB value is used as the prefix to retrieving the database config from global.conf.

**Pull request merged**